pymess/models/emails.py

```diff
@@ -63,6 +63,7 @@ class EmailMessage(BaseMessage):
     sender = models.EmailField(verbose_name=_('sender'), blank=False, null=False)
     sender_name = models.CharField(verbose_name=_('sender name'), blank=True, null=True, max_length=250)
     subject = models.TextField(verbose_name=_('subject'), blank=False, null=False)
+    pre_header = models.CharField(verbose_name=_('pre header'), blank=True, null=True, max_length=100)
```

Comment:
Only for saving to DB. For our purposes it will be sent through the template.

Reply:
What's point of having it here, if there is no use for it.

Reply:
To view in admin as a single attribute